sdk/test-utils/recorder/README.md

```diff
@@ -45,74 +45,97 @@ tests of an sdk.
 
 ## Key concepts
 
-- To **record** means to intercept any HTTP request, store it in a file, then
-  store the response received from the live resource that was originally
+- To **record** means to intercept any HTTP request (specifically, every
+  http.request call and browser's XMLHttpRequest calls), store it in a file,
+  then store the response received from the live resource that was originally
   targeted. We use a couple of mechanisms to do this for NodeJS and the
   browser. The output files are stored in `recordings/node/*` and in
   `recordings/browser/*`, which are relative to the root of the project you're
-  be working on.
+  working on.
 - To **playback** means to intercept any HTTP request and to respond it with the
   stored response of a previously recorded matching request.
+- **Sensitive information** means content that should not be shared publicly.
+  Content like passwords, unique identifiers or personal information should be
+  cleaned up from the recordings. Some functionality is provided to fix this
+  problem. You can read more at [Securing sensitive data](#securing-sensitive-data).
 
 ## Getting started
 
-We're about to go through how to set up your project to use the test-utils-recorder.
-We'll be using git and [rush](https://rushjs.io). We understand that both tools
-can be challenging to use, but we expect you to know how to use them by the
-time you decide to use this package. If you encounter any problem with these
-tools, please make sure to check at their documentation first, but we're also
-happy to answer any question! We appreciate that you're reading this 💙
+We're about to go through how to set up your project to use the
+`@azure/test-utils-recorder` package.
+
+This document assumes familiarity with [git](https://git-scm.com) and [rush](https://rushjs.io).
```

Comment:
Maybe add links to the rush intro from the root folder's readme and to the template SDK to let users know how the SDK should look like.